WSL/wsl-config.md

```diff
@@ -249,6 +244,15 @@ These settings are opt-in previews of experimental features that we aim to make
 |:----|:----|:----|:----|
 |`autoMemoryReclaim`| string | disabled | Automatically releases cached memory after detecting idle CPU usage. Set to `gradual` for slow release, and `dropcache` for instant release of cached memory. |
 |`sparseVhd`| bool | false | When set to true, any newly created VHD will be set to sparse automatically. |
+|useWindowsDnsCache**| bool | false | Only applicable when `experimental.dnsTunneling` is set to true. When this option is set to false, DNS requests tunneled from Linux will bypass cached names within Windows to always put the requests on the wire. |
```

Comment:
should we change "experimental.dnsTunneling" to "wsl2.dnsTunneling"? same for the lines below

Reply:
I'm confused by this, why should this be under `wsl2.dnsTunneling`? Isn't it an experimental flag and under `experimental.dnsTunneling`??

Reply:
dnsTunneling is not exclusive to [experimental], it can be specified directly under [wsl2]

Reply:
and yep, fixing the text to not specify 'experimental' for dnsTunneling